src/Components/Home.js

```diff
@@ -0,0 +1,316 @@
+  logoGym.src = '../img/logo.png';
+  // Create logout icon
+  const logOut = document.createElement('img');
+  logOut.src = '../img/power-off.png';
+  logOut.setAttribute('class', 'log-out');
+  headerFeed.append(logoGym, logOut);
+  // Creates main
+  const mainContainer = document.createElement('main');
+  mainContainer.setAttribute('class', 'main-feed');
+  // container Hello
+  const containerHello = document.createElement('div');
+  containerHello.setAttribute('class', 'containerhello-div');
+  // Create Div feed tittle and icon
+  const helloDiv = document.createElement('div');
+  helloDiv.setAttribute('class', 'hello-div');
+  // Create text for home view
+  const welcomeTxt = document.createElement('div');
+  welcomeTxt.textContent = 'Hello';
+  welcomeTxt.setAttribute('class', 'feed-title');
+  // IMG icon
+  const iconGym = document.createElement('img');
+  iconGym.setAttribute('class', 'icon-feed');
+  iconGym.src = '../img/icon.png';
+  // Text
+  const questionText = document.createElement('div');
+  questionText.setAttribute('class', 'question-feed');
+  questionText.textContent = 'What´s going on?';
+  helloDiv.append(welcomeTxt, iconGym);
+  containerHello.append(helloDiv, questionText);
+  // feed Container
+  const feedContainer = document.createElement('div');
+  feedContainer.setAttribute('id', 'feed');
+
+  //                          NEW POST
+  // Div new Post
+  const postDiv = document.createElement('div');
+  postDiv.setAttribute('class', 'post-input');
+  // Div input
+  const inputDiv = document.createElement('div');
+  inputDiv.setAttribute('class', 'input-feed');
+  // Img input
+  const inputImg = document.createElement('img');
+  inputImg.src = '../img/user-img.png';
+  inputImg.setAttribute('class', 'userimg-feed');
+  // input
+  const createPost = document.createElement('textarea');
+  createPost.setAttribute('cols', '40');
+  createPost.setAttribute('rows', '2');
+  createPost.setAttribute('placeholder', 'Ask to your partner...');
+  createPost.setAttribute('class', 'newpost-feed');
+  createPost.setAttribute('id', 'inputpost-feed');
+  // icon
+  const writeIcon = document.createElement('img');
+  writeIcon.src = '../img/write-icon.png';
+  writeIcon.setAttribute('class', 'writeicon-feed');
+  inputDiv.append(inputImg, createPost, writeIcon);
+  // Container button
+  const containerButton = document.createElement('div');
+  containerButton.setAttribute('class', 'containerbutton-feed');
+  // Msg Error
+  const msgError = document.createElement('div');
+  msgError.setAttribute('class', 'msgerror-feed');
+  msgError.textContent = 'Please, write something';
+  // Container divs post
+  const containerDivs = document.createElement('div');
+  containerDivs.setAttribute('class', 'container-posts');
+  // share button
+  const shareButton = document.createElement('button');
+  shareButton.setAttribute('class', 'sharebutton-feed');
+  shareButton.textContent = 'Share';
+
```

Comment:
esto podría estar en otro archivo
NewPost.js
const NewPost =()=>{
}
```suggestion
const newPost = NewPost()
fatherOfAll.apend(newPost)
```